migrate.py

```diff
@@ -54,6 +54,7 @@
 RAW_STR_TMPL = "r'''{str_val}'''"
 STR_TMPL = "'''{str_val}'''"
 
+BAD_EXT = frozenset(set(('.pyo', '.pyc')))
```

Comment:
You're doing 3 data structure conversions here. Let's simplify this:
```suggestion
BAD_EXT = frozenset({'.pyo', '.pyc'})
```